### ISSUE TITLE
An error occurred while loading the list of offers

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#9316]
Hi,

I am testing on Allegro sandbox and I got this error message when I try to view the offers:
![image](https://github.com/allegro/allegro-api/assets/147524238/4570fa04-aca1-4a5f-a339-7132b62524f1)

Our account name on sandbox is "mergado_sandbox".

Can you please have a look at it?

All the best,

Dominika

**Comment by PrzemyslawLukanowski** [comment]
Hi, I've forwarded this issue to the team responsible for this view. I'll be back with an answer.

**Comment by PrzemyslawLukanowski** [comment]
@dominikahr Did you change the offer title as we mentioned in the #9202?

**Comment by dominikahr** [comment]
No, it worked without changing the name and I am not sure what offer titile should I change. Can you please send me the EAN together with the reference number of the offer?

**Comment by PrzemyslawLukanowski** [comment]
The EAN is 5903148928791 and external.id is 1002. If the problem persists, please send us a screenshot of the information in the Console.

**Comment by dominikahr** [comment]
@PrzemyslawLukanowski We have checked, but we do not send any offer with this ID and EAN code.

**Comment by PrzemyslawLukanowski** [comment]
The offer ID is 7765818540 and is currently in the ENDED status. If you cannot find this, please send me a screenshot of what the Console returns.

**Comment by dominikahr** [comment]
Can you please explain more in detail what do you mean by "screenshot of what the Console returns"? 
We are not sending the offer to Allegro anymore, I guess I used the mentioned product for testing in the past.

**Comment by PrzemyslawLukanowski** [comment]
When you are on the My Assortment tab, please click on the right mouse button, choose 'Inspect' and then the 'Console' tab, like this:

<img width="1512" alt="Zrzut ekranu 2024-07-1 o 11 50 40" src="https://github.com/allegro/allegro-api/assets/51473661/bbaf981b-0ce2-4dd9-b51f-3ed6221b1123">

**Comment by dominikahr** [comment]
Ah sure, here you go: 
![image](https://github.com/allegro/allegro-api/assets/147524238/f01e0707-4683-4b8d-b7a6-8fb75403a1ad)

**Comment by dominikahr** [comment]
@PrzemyslawLukanowski any news? :)

**Comment by PrzemyslawLukanowski** [comment]
Could you please send also a screenshot from the "Network" tab?

**Comment by dominikahr** [comment]
Seems to be working today for some reason, but thank you anyway :)

**Comment by PrzemyslawLukanowski** [comment]
Ok, thank you for the update :)
